### ISSUE TITLE
Image: justifyContent: 'flex-end' doesn't apply layout to children

### ISSUE THREAD

**Issue by luisfcofv** [necolas/react-native-web#226]
**What is the current behavior?**

justifyContent: 'flex-end' is not working.

```
import React from 'react';
import { StyleSheet, Text, View, Image } from 'react-native';

const styles = StyleSheet.create({
  image: {
    flex: 1,
    flexDirection: 'column',
    justifyContent: 'flex-end',
  },
  textContainer: {
    backgroundColor: 'rgba(255, 255, 255, 0.7)',
    margin: 20,
    padding: 20,
  },
  breadcrumbText: {
    fontSize: 12,
  },
  featureText: {
    fontSize: 24,
  },
});

function App() {
  const imageUrl = 'https://c2.staticflickr.com/8/7347/16414056445_a776094048_b.jpg';
  return (
    <Image style={styles.image} source={{ uri: imageUrl }} resizeMode="cover">
      <View style={styles.textContainer}>
        <Text style={styles.breadcrumbText}>
          BREADCRUMB TEXT
        </Text>
        <Text style={styles.featureText}>
          FEATURE TITLE
        </Text>
      </View>
    </Image>
  );
}

export default App;
```

**What is the expected behaviour?**

The content should be placed at the end.

**Environment (include versions)**

OS: Sierra
Device: Pro
Browser: Firefox 49
React Native for Web (version): 0.0.47
React (version): 15.3.2

Current:

<img width="780" alt="screen shot 2016-10-17 at 12 03 20" src="https://cloud.githubusercontent.com/assets/1128284/19438221/4ff276e4-9468-11e6-8dc7-21927a744e71.png">

Expected:

![screen shot 2016-10-17 at 12 03 21](https://cloud.githubusercontent.com/assets/1128284/19438219/4d1f9a96-9468-11e6-9f70-21214897b18d.png)

**Comment by luisfcofv** [comment]
I managed to fix it by using a view inside the image and setting the style in there.

```
<Image style={styles.image} source={{ uri: imageUrl }} resizeMode="cover">
  <View style={styles.newContainerWithFlexColumn}>
      <View style={styles.textContainer}>
...
      </View>
  </View>
</Image>
```

**Comment by necolas** [comment]
Sound be fixed in 0.0.50
